### ISSUE TITLE
invalid memory address or nil pointer dereference while doing "drive init"

### ISSUE THREAD

**Issue by vrusinov** [odeke-em/drive#129]
Fresh installation:

```bash
$ go version
go version go1.4.2 linux/amd64
```

```bash
$  drive version
drive version 0.1.5
```

```bash
$ drive init  
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0xa1 pc=0x492843]

goroutine 1 [running]:
github.com/odeke-em/drive/src.New(0x4c20803c5c0, 0x0, 0x0)
        /<redacted>/opt/go/src/github.com/odeke-em/drive/src/commands.go:107 +0x293
main.(*initCmd).Run(0x9df118, 0x4c20800a010, 0x0, 0x0)
        /<redacted>/opt/go/src/github.com/odeke-em/drive/cmd/drive/main.go:115 +0x53
github.com/rakyll/command.Run()
        /<redacted>/opt/go/src/github.com/rakyll/command/command.go:155 +0x99
github.com/rakyll/command.ParseAndRun()
        /<redacted>/opt/go/src/github.com/rakyll/command/command.go:162 +0x20
main.main()
        /<redacted>/opt/go/src/github.com/odeke-em/drive/cmd/drive/main.go:65 +0x3714

goroutine 5 [syscall]:
os/signal.loop()
        /usr/lib/google-golang/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
        /usr/lib/google-golang/src/os/signal/signal_unix.go:27 +0x35

goroutine 6 [chan receive]:
main.func·001()
        /<redacted>/opt/go/src/github.com/odeke-em/drive/cmd/drive/main.go:797 +0x64
created by main.initContext
        /<redacted>/opt/go/src/github.com/odeke-em/drive/cmd/drive/main.go:802 +0x2e3```

**Comment by odeke-em** [comment]
Thank you for reporting this, would you mind giving me some diagnostic information by running the following:
```shell
$ cd $GOPATH/src/github.com/odeke-em/drive
$ git log | head -n 20
```

**Comment by odeke-em** [comment]
I am on
`go version go1.4.1 darwin/amd64`
and it works alright for me.

**Comment by l3iggs** [comment]
I use `go version go1.4.2 linux/amd64` and `drive version 0.1.5` and I don't get any errors with `drive init`.

**Comment by jorgesca** [comment]
I had the same error message earlier today when I installed drive. Solved it by following the guidelines related to setting GOPATH [1] in the Arch wiki, and re-installing drive afterwards. I just installed go as a dependency for drive, so I hadn't set anything related to it. Maybe the same applies to the OP?

1- https://wiki.archlinux.org/index.php/Go#.24GOPATH

**Comment by odeke-em** [comment]
If it isn't related to cgo just tripping out, please pull again from master https://github.com/odeke-em/drive/pull/131/files could have potentially fixed it.

**Comment by vrusinov** [comment]
Sorry for late reply, I got this only on one machine and it was not reachable on weekend.

Looks like it fixed in 0.1.6, I've updated and can no longer reproduce it.

Thanks!
